examples/src/main/java/com/example/CommonsTextFuzzer.java

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 Code Intelligence GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.example;
+
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
+import org.apache.commons.text.StringSubstitutor;
+import com.code_intelligence.jazzer.api.Jazzer;
+
+public class CommonsTextFuzzer {
+    public static void fuzzerTestOneInput(FuzzedDataProvider data) {
+        try {
+            StringSubstitutor.createInterpolator().replace(data.consumeAsciiString(20));
```

Comment:
`data.consumeRemainingAsAsciiString()` would be slightly more idiomatic. Do you know whether that makes the bug harder to find?

Reply:
I did a couple of runs and it seemed that convergence was better with a limited length, but again, I will try and do some measurements if that's interesting.

Reply:
Yeah, this seems to make a big difference in runtime as far as I could see. I am not sure exactly why, I don't know nearly enough about the internals, but my educated guess was that because it has to fuzz `${script:js:(...)}` it is best to minimize the search space (the `js:(...)` part is injected by the hook, but the rest has to be found by the fuzzer and every input that doesn't end with a `}` won't cut it, no matter that it begins correctly.